src/main/java/org/spongepowered/api/service/economy/EconomyService.java

```diff
@@ -0,0 +1,168 @@
+     * only. See {@link Account#getOwner()} for
+     * reference.</p>
+     * 
+     * @param owner the owner string to search with
+     * 
+     * @return the Account owned by the given owner, or
+     *         Optional.absent() otherwise
+     * 
+     * @see Account#getOwner()
+     * @see EconomyService#createNewAccount(String)
+     */
+    Optional<Account> getAccount(String owner);
+    
+    /**
+     * Creates a new {@link Account} and assigns it to the
+     * given owner string.
+     * 
+     * <p>This method cannot override existing Accounts. If
+     * the Account already exists, this method will return
+     * Optional.absent() to indicate that creation has
+     * failed. Existing Accounts should be reset before
+     * they are replaced.</p>
+     * 
+     * <p>Owner strings are not representative of players
+     * only. See {@link Account#getOwner()} for
+     * reference.</p>
+     * 
+     * @param owner the owner string to assign the new
+     *        Account to
+     * 
+     * @return whether or not the Account was created
+     *         successfully
+     * 
+     * @see Account
+     * @see Account#getOwner()
+     * @see EconomyService#resetAccount(String)
+     */
+    Optional<Account> createNewAccount(String owner);
+    
+    /**
+     * Retrieves the {@link Account} denoted by
+     * the given {@link UUID}, belonging to a {@link Player}
+     * , in the form given by {@link UUID#toString()}.
+     * Shortcut method for {@link #getAccount(String)}.
+     * 
+     * @param uuid the UUID of the Player owning the Account
+     * 
+     * @return the Player Account found with the given name,
+     *         or Optional.absent() otherwise
+     * 
+     * @see Account
+     * @see EconomyService#getAccount(String);
+     */
+    Optional<Account> getPlayerAccount(UUID uuid);
+    
+    /**
+     * Retrieves the {@link Account} denoted by
+     * {@link Account#CUSTOM_DATA_PREFIX} + the given
+     * account name string. Shortcut method for
+     * {@link #getAccount(String)}.
+     * 
+     * @param accountName the name of the custom account
+     * 
+     * @return the custom Account found with the given name,
+     *         or Optional.absent() otherwise
+     * 
+     * @see Account
+     * @see EconomyService#getAccount(String);
+     */
+    Optional<Account> getCustomAccount(String accountName);
+    
+    /**
+     * Retrieves the {@link Account} assigned to the
+     * server. Shortcut method for
+     * {@link #getAccount(String)}.
+     * 
+     * @return the Account belonging to the server
+     * 
+     * @see Account
+     * @see EconomyService#getAccount(String);
+     */
+    Account getServerAccount();
+    
+    /**
+     * Returns an immutable list of all {@link Account}s
+     * belonging to a {@link Player}.
+     * 
+     * <p>This method should try to return all Accounts
+     * registered belonging to Players; including offline
+     * Players.</p>
```

Comment:
Just say `belonging to a {@link User}` above, which includes online and offline players.